Explicit dependency on TIFF libraries, so that builds succeed on macOS.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -36,7 +36,7 @@ set(CMAKE_C_STANDARD 99)
 ecbuild_add_option( FEATURE CAIRO
                     DEFAULT ON
                     DESCRIPTION "cairo support[png/jpeg]"
-                    REQUIRED_PACKAGES PangoCairo)
+                    REQUIRED_PACKAGES PangoCairo TIFF)
 
 ecbuild_add_option( FEATURE DOCUMENTATION
                     DEFAULT OFF
@@ -233,6 +233,7 @@ endif()
 
 if( HAVE_CAIRO )
   list( APPEND MAGICS_PRIVATE_INCLUDE_DIRS ${PANGOCAIRO_INCLUDE_DIRS})
+  list( APPEND MAGICS_EXTRA_LIBRARIES     ${TIFF_LIBRARIES})
 
   if( EC_OS_NAME MATCHES "windows" )
 
